v0/destinations/customerio/transform.js

```diff
@@ -220,6 +220,15 @@ function processSingleMessage(message, destination) {
       throw new Error(`could not determine type ${messageType}`);
   }
   const response = responseBuilder(message, evType, evName, destination);
+
+  // replace default domain with EU data center domainc for EU based account
+  if (destination.Config.datacenterEU) {
+    response.endpoint = response.endpoint.replace(
+      "track.customer.io",
+      "track-eu.customer.io"
+    );
+  }
```

Comment:
Instead of replacing the `endpoint` here, IMO it's better to assign the value in `responseBuilder` method based on the destination config.

Reply:
I’ve update the code, but when I perform test it seems that previous code takes less than 1 sec to complete the test cases where as new code takes around 1.2 sec.

Reply:
Commit reverted